### ISSUE TITLE
Custom currency

### ISSUE THREAD

**Issue by SkootyPex** [BrendonCurmi/FusionPixelmon#66]
is it possible to make it so the currency used in /pokedesigner is a custom currency, ive made a currency called "Gold" with the totaleconomy plugin but when changing "pokedollars" to gold in the config nothing changes

**Comment by BrendonCurmi** [comment]
Yes, on the Sponge version you can add custom currencies. The currencies need to be configured through the economy plugin (I've tested with economylite) and then the identifier specified in the fusionpixelmon config. You need to make sure it's setup correctly with the economy plugin, and the correct identifier is used (whether that's the currency id, name, etc), test if unsure. Remember to restart the server every time you update the fusionpixelmon config
